hack/update-vendor-licenses.sh

```diff
@@ -152,19 +152,25 @@ cd "${LICENSE_ROOT}"
 
 VENDOR_LICENSE_FILE="Godeps/LICENSES"
 TMP_LICENSE_FILE="/tmp/Godeps.LICENSES.$$"
+
+VENDOR_LICENSE_DIR="Godeps/ALL_LICENSES"
+TMP_VENDOR_LICENSE_DIR="/tmp/Godeps.LICENSES.DIR.$$"
+
+if [ ! -d ${VENDOR_LICENSE_DIR} ] ; then
```

Comment:
nit-picking.  As a point of habit, we almost always quote variables where they must expand to a single value: `"${VENDOR_LICENSE_DIR}"`